mpb/mpb.c

```diff
@@ -494,10 +497,15 @@ void init_params(integer p, boolean reset_fields)
      }
 
      mpi_one_printf("%d k-points:\n", k_points.num_items);
-     for (i = 0; i < k_points.num_items; ++i)
+     for (i = 0; i < MIN(k_points.num_items, MAX_KPOINTS_PRINT); ++i)
```

Comment:
Probably set
```c
int max_kpoints_print = verbosity >= 2 ? k_points.num_items : MIN(k_points.num_items, MAX_KPOINTS_PRINT);
```
so that in verbose mode it prints all of them.

Reply:
`mpb_verbosity`

Reply:
Sure, that makes sense: I've forced pushed a change to that effect.

I set the threshold at `mpb_verbosity >= 3` rather than `mpb_verbosity >= 2` though (analogously to #123; i.e. truncated listing would be default behavior).